### ISSUE TITLE
JSON replay only loads first move of game

### ISSUE THREAD

**Issue by kesslermaximilian** [Hanabi-Live/hanabi-live#2825]
The site offers the functionality of uploading a JSON (in the format of https://raw.githubusercontent.com/Hanabi-Live/hanabi-live/main/misc/example_game_with_comments.json) and watch the corresponding replay. When creating such a replay, only the first move actually appears in the replay, the rest seems to be ignored.

I tested this with:
- qutebrowser and ungoogled chromium
- both solo / shared replays
- the example json from the above link (without comments) [as well as a JSON i wanted to upload and a couple others, but don't think it matters here)

**Comment by kesslermaximilian** [comment]
Okay, after trying around a bit more, I found that I specified the targets of play/discard actions with the indices in the current hand, not in the initial deck, so my JSON was just flawed and the error is mine.
Would have liked some error message from the site to tell me that the JSON was not valid, instead of just throwing away the corresponding moves.

I still don't know what is wrong with the example game (or whether I messed up there, too), but for now, I'm fine since I resolved my issue.
